src/neptune/new/internal/disk_queue.py

```diff
@@ -139,12 +139,13 @@ def _get_serialized(self) -> Optional[dict]:
         return _json
 
     def get_batch(self, size: int) -> Tuple[List[T], int]:
-        first, ver = self.get()
-        if not first:
-            return [], ver
+        serialized_first = self._get_serialized()
```

Comment:
`self.get()` was doing something more. There is logic for skipping first X operations if processing was resumed and some ops was already acked. Now this logic is missing.